VideoLocker/res/layout/fragment_discussion_topics.xml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    xmlns:tools="http://schemas.android.com/tools"
     android:background="@color/white"
+    android:focusable="true"
+    android:focusableInTouchMode="true"
```

Comment:
This is the wrong approach in my opinion, with non-intended side-effects. All of these changes can be replaced with a simple `discussionTopicsSearchView.clearFocus();` in the `onResume()` callback (this would need to be deferred until the layout stage though):

```` java
@Override
public void onResume() {
    super.onResume();
    discussionTopicsSearchView.post(new Runnable() {
        @Override
        public void run() {
            discussionTopicsSearchView.clearFocus();
        }
    });
}
````